source/tmpl/facewall/filters_age.tmpl

```diff
@@ -2,23 +2,23 @@
 <fieldset class="ns_filter__control ns_filter__control--age ns_filter__control--active">
     <legend>By age</legend>
     <div>
-        <label for="ns_facewall__input--age-0">17-20</label>
-        <input class="ns_facewall__input ns_facewall__input--age-17-20" value="age-17-20" type="checkbox" />
+        <label for="ns_facewall__input--age-0">15-20</label>
+        <input id="ns_facewall__input--age-0" class="ns_facewall__input ns_facewall__input--age-15-20" value="age-15-20" type="checkbox" />
     </div>
```

Comment:
...comment on individual line numbers. Good practice to do that rather than copy and paste code into comments.